### PR TITLE
fix(e2e): ingestion related flaky failures

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes:

Fixes <issue-number>

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by karanh37 — DISMISSED** [review]

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by akash-jain-10 — DISMISSED** [review]

**Review by ShaileshParmar11 — DISMISSED** [review]

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/dataInsightApp.ts [hunk 1]
```diff
@@ -115,7 +112,7 @@ setup(
               }),
         }
       )
-      .toBe('success');
+      .not.toEqual(/(running|queued)/);
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/dataInsightApp.ts:116]: after this, i think we need to add a success check as well, else we won't know what the status of the pipeline

File: openmetadata-ui/src/main/resources/ui/playwright/support/entity/ingestion/ServiceBaseClass.ts [hunk 2]
```diff
@@ -262,8 +262,8 @@ class ServiceBaseClass {
           intervals: [30_000, 15_000, 5_000],
         }
       )
-      // To allow partial success
-      .toContain('success');
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/playwright/support/entity/ingestion/ServiceBaseClass.ts:266]: same here

Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/playwright/support/entity/ingestion/ServiceBaseClass.ts:266]: It's contain so partialSucess will work too right

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/dataInsightApp.ts (hunk 1 above)
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/playwright/e2e/dataInsightApp.ts:116]: We are checking for success in next line. After a page reload

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by ShaileshParmar11 — DISMISSED** [review]

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by akash-jain-10 — APPROVED** [review]

**Pull request merged**